### PR TITLE
:art: refactor patient card

### PR DESCRIPTION
**Pull request by issa-diallo**

### CODE REVIEW

File: src/components/PatientLine.tsx [hunk 1]
```diff
@@ -1,49 +1,106 @@
...
+      </Card>
+
+      <Modal show={buttonsModalShow} onHide={handleCloseButtonsModal}>
+        <Modal.Header closeButton>
+          <Modal.Title>{t('text.actionsPatients')}</Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+          <Button size='sm' variant='dark' className='me-2' onClick={goToPatient}>
+            {t('text.seePatient')}
+          </Button>
```
Comment (AndreMiras) [inline src/components/PatientLine.tsx:91]: To me this shouldn't be part of this popup, but accessible via a simple click on the card

File: src/components/PatientLine.tsx [hunk 2]
```diff
@@ -1,49 +1,106 @@
...
-    </Card>
+    <>
+      <Card className='shadow mt-4'>
+        <Card.Body>
+          <div className='row align-items-start'>
+            <div className='col'>
+              <div className='d-flex align-items-center'>
+                <Badge bg='secondary' text='dark' className='me-2'>
+                  {patient.lastname} {patient.firstname}
+                </Badge>
```
Comment (AndreMiras) [inline src/components/PatientLine.tsx:66]: Why making the patient name so small on a badge? I think this is something that should standout, not be small

**Review by AndreMiras — APPROVED** [review]
Left a couple of UX comments, but looks OK overall

File: src/components/PatientLine.tsx (hunk 2 above)
Comment (issa-diallo) [reply src/components/PatientLine.tsx]: Yes I agree I going to change

File: src/components/PatientLine.tsx (hunk 1 above)
Comment (issa-diallo) [reply src/components/PatientLine.tsx:89]: oki I going to change

**Pull request merged**